ui/src/app/components/runtime-configuration-panel.rtl.spec.tsx

```diff
@@ -1046,38 +946,37 @@ describe(RuntimeConfigurationPanel.name, () => {
 
   it('should allow creation with defaults when no runtime exists', async () => {
     setCurrentRuntime(null);
-
+    mockUseCustomRuntime();
     component();
 
-    clickExpectedButton('Create');
-
-    await waitFor(() => {
-      expect(runtimeApiStub.runtime.status).toEqual('Creating');
-      expect(runtimeApiStub.runtime.gceWithPdConfig.machineType).toEqual(
-        'n1-standard-4'
-      );
-      expect(runtimeApiStub.runtime.gceConfig).toBeUndefined();
-      expect(runtimeApiStub.runtime.dataprocConfig).toBeUndefined();
-    });
-  });
-
-  it('should show customize after create', async () => {
-    setCurrentRuntime(null);
+    await clickExpectedButton('Create');
 
-    component();
+    await waitFor(
+      async () => {
+        expect(mockSetRuntimeRequest).toHaveBeenCalledTimes(1);
+      },
+      { timeout: 5000 }
+    );
 
-    clickExpectedButton('Create');
+    expect(
+      mockSetRuntimeRequest.mock.calls[firstCall][firstParameter].runtime
+        .gceWithPdConfig.machineType
+    ).toEqual('n1-standard-4');
 
-    // creation closes the panel. re-render with the new runtime state
-    await waitFor(() => {
-      component();
+    expect(
+      mockSetRuntimeRequest.mock.calls[firstCall][firstParameter].runtime
+        .gceConfig
+    ).toBeUndefined();
 
-      // now in Customize mode
+    expect(
+      mockSetRuntimeRequest.mock.calls[firstCall][firstParameter].runtime
+        .dataprocConfig
+    ).toBeUndefined();
 
-      const button = screen.getByRole('button', { name: 'Customize' });
-      expect(button).toBeInTheDocument();
-      expectButtonElementEnabled(button);
-    });
+    // await waitFor(() => {
```

Comment:
should this be removed?

Reply:
Yes, this is duplicative of the two assertions above.